backend/server.js

```diff
@@ -32,35 +32,49 @@ const createWorker = (path: string): {
   rpcSbp: (...args: any) => Promise<any>,
   terminate: () => Promise<number>
 } => {
-  const worker = new Worker(path)
-  const ready = new Promise((resolve, reject) => {
-    worker.on('message', (msg) => {
-      if (msg === 'ready') resolve()
-    })
-    worker.on('error', reject)
-  })
+  let worker
+  let ready: Promise<void>
 
-  const rpcSbp_ = (...args: any) => {
+  const launchWorker = () => {
+    worker = new Worker(path)
     return new Promise((resolve, reject) => {
+      worker.on('message', (msg) => {
+        if (msg === 'ready') {
+          worker.off('error', reject)
+          worker.on('error', (e) => {
+            console.error(e, 'Error on worker', path)
```

Comment:
Could you update this error message to say: `Running worker ${filename(path)} terminated. Attempting relaunch...`